### ISSUE TITLE
In-Unit Mechanic Apprenticeship education lasts until student is 9999 

### ISSUE THREAD

**Issue by ussdefiant60** [MegaMek/mekhq#5058]
### Prerequisites and Pre-Issue Checklist

- [X] I'm reporting the issue to the correct repository:
- [MegaMek](https://github.com/MegaMek/megamek/issues)
- [MegaMekLab](https://github.com/MegaMek/megameklab/issues)
- [MekHQ](https://github.com/MegaMek/mekhq/issues)

- [X] I've tested the issue against at least the latest MILESTONE version
- [X] I've asked on the MegaMek Discord about the error
- [X] I've reviewed the BattleTech rules and MekHQ documentation, and I've confirmed that something isn't working as intended.
- [X] I've searched the Github tracker and haven't found the issue listed

### Severity *

Medium (Gameplay Limitation): Non-core functionality is impaired, providing a suboptimal but playable experience.

### Brief Description *

Simply put, In-Unit Mechanic Apprenticeship seems to be messed up somehow, as per the attached screenshot. @IllianiCBT asked that i toss it up here so he'd remember to fix it.

### Operating System *

Windows 11

### Java Version *

Java 17

### MekHQ Suite Version *

Free Text (type manually)

### Custom MekHQ Version

50.01 NIGHTLY 1647

### Attach Files

https://cdn.discordapp.com/attachments/902192028706807828/1296634413697994783/image.png?ex=67130066&is=6711aee6&hm=dfbb7e1d90729e4d4585127cd82b352711602d57c6c9236e7280949b8e3da1fc&

### Final Checklist

- [X] I've checked to make sure that this issue has not already been filed
- [X] I'm reporting only one issue in this ticket for clarity and focus

**Comment by IllianiCBT** [comment]
Closing as resolved
